protocols/gossipsub/CHANGELOG.md

```diff
@@ -2,8 +2,11 @@
 
 - Signed messages now use sequential integers in the sequence number field.
   See [PR 3551].
+- Gracefully disable handler on stream errors. Deprecate a few variants of `HandlerError`.
+  See [PR 3625].
 
-[PR 3551]: https://github.com/libp2p/rust-libp2p/pull/3551
+[PR 3551]: https://github.com/libp2p/rust-libp2p/pull/3551git pull
```

Comment:
```suggestion
[PR 3551]: https://github.com/libp2p/rust-libp2p/pull/3551
```

Reply:
yes, this was an accidental focus switch :(